packages/backend/src/database/migrations/20240604140542_add_missing_indexes_for_dashboards_query.ts

```diff
@@ -0,0 +1,46 @@
+import { Knex } from 'knex';
+
+const newTableSingularIndexes = {
+    dashboard_versions: ['dashboard_id', 'updated_by_user_uuid'],
+    spaces: ['project_id'],
+    projects: ['organization_id'],
+    pinned_dashboard: ['pinned_list_uuid', 'dashboard_uuid'],
+    dashboard_tiles: ['dashboard_tile_uuid', 'dashboard_version_id'],
+    dashboard_tile_charts: ['dashboard_tile_uuid', 'saved_chart_id'],
```

Comment:
https://github.com/lightdash/lightdash/blob/67db14c6c9601a984c2aa7643b4dca501f207a50/packages/backend/src/models/DashboardModel/DashboardModel.ts#L349

https://github.com/lightdash/lightdash/blob/67db14c6c9601a984c2aa7643b4dca501f207a50/packages/backend/src/models/DashboardModel/DashboardModel.ts#L355

Reply:
Same as before, L349-L350 should use the already existing two-column index on `dashboard_version_id, dashboard_tile_uuid`

The `saved_chart_id` index is needed 👍